### PR TITLE
Add various Fable-related podcast episodes

### PR DESCRIPTION
**Pull request by StachuDotNet**
Adds 3 podcast episodes.

- Hanselminutes on SAFE stack
- Hanselminutes on Fable
- WTF# on Fable

### CODE REVIEW

**Comment by StachuDotNet** [comment]
I think that the test is bad :)

**Comment by kunjee17** [comment]
@StachuDotNet there must be some link issue... let me have a look... there is a duplicate link there.

File: README.md [hunk 1]
```diff
@@ -76,10 +76,13 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
...
     * [FableConf 2017 videos playlist](https://www.youtube.com/watch?v=ssKX7T3lNvw&list=PL4vOF1DmOhsl8mXF8NkR_rgkVJP2nR4Mh) - All FableConf 2017 videos
     * [Fable conference talks videos playlist](https://www.youtube.com/watch?v=iP-50fj06Eo&list=PL4vOF1DmOhsnIq6db_j3jv29HoatR4G-Q) - Other Fable talks on Youtube
+* [Conquer the JavaScript Ecosystem with F# and Fable](https://skillsmatter.com/skillscasts/9732-conquer-the-javascript-ecosystem-with-f-sharp-and-fable-audience-level-beginner) - A gentle introduction of Fable from creator _Alfonso Garcia-Caro_
+* [WTF# is Fable?](https://wtfsharp.net/wtf-is-fable) - An overview of the Fable ecosystem on the [WTF#](https://wtfsharp.net) podcast.
+* [From F# to JavaScript and beyond with Fable](https://hanselminutes.com/616/from-f-to-javascript-and-beyond-with-fable-and-alfonso-garcia-caro) - with Alfonso Garcia-Caro on Scott Hanselman's "[Hanselminutes](https://hanselminutes.com)" podcast
+* [F# and the SAFE stack](https://hanselminutes.com/624/f-and-the-functional-safe-stack-with-krzysztof-cielak) - with Krzysztof Cieślak on Scott Hanselman's "[Hanselminutes](https://hanselminutes.com)" podcast
```
Comment (kunjee17) [inline README.md]: @StachuDotNet there are two multiple links pointing to hanselminutes.com ...

**Comment by kunjee17** [comment]
Build issue solved. 👍

**Pull request merged**